RELEASE/scripts/autoscend/combat/auto_combat_default_stage3.ash

```diff
@@ -69,6 +69,12 @@ string auto_combatDefaultStage3(int round, monster enemy, string text)
 		}
 	}
 
+	//delevel if you have a loofah lei
```

Comment:
on line 174 of this file (`if(enemy_la <= 150)`) is where there's a check to make sure enemy can be staggered. Would you only want to use this skill if the enemy can actually get staggered? If yes, it should get moved down to that check

Reply:
That makes sense.

Reply:
Then again, deleveling is nice too.  Maybe a sufficient delevel will prevent the +151ML foe from hitting you.  ... I think I'll take your suggestion anyway, maybe fine-tune it later if users need it.